src/gfx/qt.cpp

```diff
@@ -808,9 +808,11 @@ QImageReader *GfxProcQT::readbitmapLibraw(int &w, int &h, int &orientation, QStr
               << " " << imgdata.sizes.height
               << " " << imgdata.thumbnail.twidth
               << " " << imgdata.thumbnail.theight
+              << " " << imgdata.thumbnail.tlength
               << " " << imgdata.sizes.flip;
 
-    if (imgdata.thumbnail.twidth > 0 && imgdata.thumbnail.theight > 0)
+    if (imgdata.thumbnail.twidth > 0 && imgdata.thumbnail.theight > 0
```

Comment:
we might want to add a LOG_warn in the else.

Reply:
I think maybe it is enough with only printing the `imgdata`, avoiding an extra log message. Thanks for the suggestion.